scriptoocloud/src/main/resources/config.properties

```diff
@@ -4,3 +4,5 @@ db.username=${db.user}
 db.password=${db.password}
 git.repositories.path = ${git.repositories.path}
 digingUrl = http://diging.asu.edu/index.html
+rootUploadLocation = uploadDirectory
```

Comment:
Is this the directory to upload files? If so, it needs to be configurable through the pom. If not, what is it used for?